docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md

```diff
@@ -0,0 +1,63 @@
+# Dictionaries in SKOS namespace
+
+ODM also supports dictionaries in [SKOS](https://www.w3.org/2009/08/skos-reference/skos.html) namespace. This feature is
+useful when you
```

Comment:
leave just 'useful to organise' and organise with 's'

Reply:
It's beyond repair here

Reply:
fixed, but not s->z